### PR TITLE
Google vision label detection

### PR DESCRIPTION
**Pull request by joncatchy**
Add example using Google Vision to detect labels in images.

### CODE REVIEW

**Comment by cbetta** [comment]
Thank you for this sample @joncatchy. Can you please add a README with instructions? Please use this README as a template:

https://github.com/box-community/sample-image-skills/tree/master/acuant-assureid-goverment-id-data-extraction

File: cloud-vision-label-detection/serverless.yml [hunk 1]
```diff
@@ -0,0 +1,42 @@
+service: box-skill # NOTE: Don't put the word "google" in here
+
+provider:
+  name: google
+  runtime: nodejs8
+  project: hip-river-225020
+  # the path to the credentials file needs to be absolute
+  credentials: ~/.gcloud/102092787bb00e64304593178318ef8e8b03fde2a.json
```
Comment (cbetta) [inline cloud-vision-label-detection/serverless.yml]: Can we make this look nicer?

Comment (joncatchy) [reply cloud-vision-label-detection/serverless.yml]: Done!

**Comment by cbetta** [comment]
Hey @joncatchy something seems to have gone a bit wrong. It seems like you've "changed" just about every file in the repo now, though mostly with no changes. It might be worth doing a fresh pull request.

**Comment by eshaverma** [comment]
hi @joncatchy  just fyi, we already did add a google vision skill, wondering if you would like to extend on that one?

**Comment by cbetta** [comment]
Closing due to inactivity.

**Pull request closed without merging**